cbt.go

```
@@ -1710,6 +1726,42 @@ func doSet(ctx context.Context, args ...string) {
 	}
 }
 
+func doReadModifyWrite(ctx context.Context, args ...string) {
+	if len(args) < 5 {
+		log.Fatalf("cbt readmodifywrite <table-id> <rowkey> <family> <qualifier> [append=<val>] [increment=<delta>]")
+	}
+	table := args[0]
+	rowkey := args[1]
+	family := args[2]
+	qualifier := args[3]
+
+	parsed, err := parseArgs(args[4:], []string{"append", "increment"})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	mut := bigtable.NewReadModifyWrite()
+	if arg, ok := parsed["append"]; ok {
+		mut.AppendValue(family, qualifier, []byte(arg))
+	} else if arg, ok := parsed["increment"]; ok {
+		delta, err := strconv.ParseInt(arg, 0, 64)
+		if err != nil {
+			log.Fatalf("While parsing increment=...: %v", err)
+		}
+		mut.Increment(family, qualifier, delta)
+	}
```

Should we throw an error when user has specified both append and increment ?